api/v01/entities/vrp_input.rb

```diff
@@ -152,6 +152,7 @@ module VrpConfiguration
   end
 
   params :vrp_request_preprocessing do
+    optional(:unconstrainted_initialization, type: Boolean, desc: 'Use experimental algorithm to find an initial solution')
```

Comment:
Why is it called `unconstrainted_initialization` when the PR calls about an initial solution to find ? 
Shouldn't it be called `initialise_solution` ?

Reply:
There are many initial solution calculation methods. This one is just a different one, which is unconstrainted... But you right, we should call it this way everywhere.